lib/aptible/cli/subcommands/operation.rb

```diff
@@ -34,6 +34,21 @@ def self.included(thor)
 
               attach_to_operation_logs(o)
             end
+
+            desc 'operation:logs OPERATION_ID', 'View logs for given operation'
+            define_method 'operation:logs' do |operation_id|
+              o = Aptible::Api::Operation.find(operation_id, token: fetch_token)
+              raise "Operation ##{operation_id} not found" if o.nil?
+
+              unless %w(succeeded failed).include? o.status
+                e = 'Error - You can view the logs when operation is complete.'
```

Comment:
In the *future* we can log the command to follow the logs here. This will likely be more of an edge case anyway so I'm not worried about making the change now.